backend/controllers/establishmentRoll/noCellAllocated.js

```diff
@@ -0,0 +1,49 @@
+const { formatName, putLastNameFirst, getTime, stripAgencyPrefix } = require('../../utils')
```

Comment:
It might be more efficient to make one call to:  `POST /api/prisoners `with ` {"offenderNos": [
    "string"
  ]}`

Reply:
Same for staff details. Use `POST /api/users/list` with ```
[
  "string"
]
```

Reply:
Good shout, forgot about that endpoint. Ta